src/main/java/twilightforest/entity/EntityTFIceExploder.java

```diff
@@ -79,8 +81,9 @@ protected void onDeathUpdate() {
 		if (this.deathTime == 60) // delay until 3 seconds
 		{
 			if (!world.isRemote) {
+				// TODO: I think this is correct
```

Comment:
not fully equivalent to the old parameter, but good enough. todo can be removed.